src/lib/classes/saml_client.php

```diff
@@ -55,6 +55,17 @@ public function authenticate()
         $username = $attrs[$this->settings->get_attribute('username')][0];
         if(get_user_by('login',$username))
         {
+            /*
```

Comment:
I got rid of the weird (and probably very insecure) use of `SAML_Client::user_password()` in https://github.com/ktbartholomew/saml-20-single-sign-on/commit/34c0d64ef0aaacd17608f7e82139d16a9afde220 which should make this entire section obsolete. I'd just remove it.

The idea is that the passwords stored in the WordPress database will be completely random, and actually unknown by the plugin. This is possible because `SAML_Client::simulate_signon()` now uses [`wp_set_auth_cookie`](https://codex.wordpress.org/Function_Reference/wp_set_auth_cookie) which doesn't require us to know the user's password.